.github/actions/scenarios/dubbo/lane-dubbo3/action.yml

```diff
@@ -0,0 +1,285 @@
+name: "Dubbo Lane Test"
+description: "Auto test for dubbo lane"
+runs:
+  using: "composite"
+  steps:
+    - name: entry
+      uses: ./.github/actions/common/entry
+      with:
+        log-dir: ./logs/dubbo-lane
+    - name: package dubbo 3.0.x tests
+      shell: bash
+      if: matrix.dubbo-version == '3-0'
+      run: mvn package -Ddubbo.version=3.0.${{ matrix.dubbo-versions }} -DskipTests --file sermant-integration-tests/dubbo-test/pom.xml
+    - name: package dubbo 3.1.x tests
+      shell: bash
+      if: matrix.dubbo-version == '3-1'
+      run: mvn package -Ddubbo.version=3.1.${{ matrix.dubbo-versions }} -DskipTests --file sermant-integration-tests/dubbo-test/pom.xml
+    - name: package dubbo 3.2.x tests
+      shell: bash
+      if: matrix.dubbo-version == '3-2'
+      run: mvn package -Ddubbo.version=3.2.${{ matrix.dubbo-versions }} -DskipTests --file sermant-integration-tests/dubbo-test/pom.xml
+    - name: post config to local-cse
+      shell: bash
+      env:
+        TEST_TYPE: lane-config
+      run: mvn test --file sermant-integration-tests/dubbo-test/pom.xml
+    - name: start provider service all
+      shell: bash
+      if: matrix.registry-model == 'all'
+      env:
+        dynamic.config.dynamicConfigType: KIE
+        dynamic.config.serverAddress: 127.0.0.1:30110
+        SERVICE_META_ENVIRONMENT: development
+        SERVICE_META_APPLICATION: lane
+        SERVICE_META_VERSION: 1.0.0
+        SERVER_PORT: 18021
+        DUBBO_PROTOCOL_PORT: 18821
+        DUBBO_REGISTRY_MODE: all
```

Comment:
try to use matrix.registry-model, and remove if

Reply:
fixed